exercise-src/react-native/06-custom-components/03-problem/src/screens/StateList/StateList.tsx

```diff
@@ -0,0 +1,5 @@
+import type { FC } from "react"
+import { View, Text } from "react-native"
+import ListItem from "./components/ListItem"
+
```

Comment:
These files are empty, but should we even provide the empty files for the exercise?

Reply:
Yeah, I think this is a good idea. I would also add a comment in this file like:

> Exercise: Add the `StateList` component implementation here.